### PR TITLE
Add ManageContent

### PR DESCRIPTION
**Pull request by cindylay**
## Category

- [x] Content fix
- [ ] New article

## What's in this Pull Request?
 
Updating auth article to reflect new container permission ManageContent

### CODE REVIEW

**Comment by cindylay** [comment]
will await approval from eng before merging

**Review by andrewconnell — APPROVED** [review]

**Pull request merged**
